src/genie/libs/parser/iosxe/tests/ShowBootflash/cli/equal/golden_output_1_expected.py

```diff
@@ -0,0 +1,174 @@
+      "file_date": "Nov 03 2020 05:54:31.0000000000 +00:00",
+      "file_name": "/bootflash/tracelogs/btrace_rotate_sh_R0-0.17615_5767.20201102174930.bin"
+    },
+    "16": {
+      "file_index": "16",
+      "file_length": "8192",
+      "file_date": "Nov 01 2020 06:40:27.0000000000 +00:00",
+      "file_name": "/bootflash/tracelogs/btrace_rotate_sh_R0-0.17615_5765.20201031194120.bin"
+    },
+    "17": {
+      "file_index": "17",
+      "file_length": "8192",
+      "file_date": "Nov 06 2020 03:06:45.0000000000 +00:00",
+      "file_name": "/bootflash/tracelogs/btrace_rotate_sh_R0-0.17615_5770.20201105160741.bin"
+    },
+    "18": {
+      "file_index": "18",
+      "file_length": "8192",
+      "file_date": "Nov 09 2020 01:44:54.0000000000 +00:00",
+      "file_name": "/bootflash/tracelogs/btrace_rotate_sh_pmanlog_R0-0.15294_2054.20200929224107.bin"
+    },
+    "19": {
+      "file_index": "19",
+      "file_length": "8192",
+      "file_date": "Oct 31 2020 07:36:22.0000000000 +00:00",
+      "file_name": "/bootflash/tracelogs/btrace_rotate_sh_R0-0.17615_5764.20201030193123.bin"
+    },
+    "20": {
+      "file_index": "20",
+      "file_length": "8192",
+      "file_date": "Nov 02 2020 05:44:33.0000000000 +00:00",
+      "file_name": "/bootflash/tracelogs/btrace_rotate_sh_R0-0.17615_5766.20201101184525.bin"
+    },
+    "21": {
+      "file_index": "21",
+      "file_length": "8192",
+      "file_date": "Oct 30 2020 08:35:45.0000000000 +00:00",
+      "file_name": "/bootflash/tracelogs/btrace_rotate_sh_R0-0.17615_5763.20201029213708.bin"
+    },
+    "22": {
+      "file_index": "22",
+      "file_length": "4096",
+      "file_date": "Apr 20 2017 23:42:59.0000000000 +00:00",
+      "file_name": "/bootflash/tracelogs/modules"
+    },
+    "23": {
+      "file_index": "23",
+      "file_length": "4096",
+      "file_date": "Apr 20 2017 23:43:56.0000000000 +00:00",
+      "file_name": "/bootflash/core"
+    },
+    "24": {
+      "file_index": "24",
+      "file_length": "4096",
+      "file_date": "Apr 20 2017 23:42:59.0000000000 +00:00",
+      "file_name": "/bootflash/core/modules"
+    },
+    "25": {
+      "file_index": "25",
+      "file_length": "1",
+      "file_date": "Nov 09 2020 01:36:15.0000000000 +00:00",
+      "file_name": "/bootflash/core/.callhome"
+    },
+    "26": {
+      "file_index": "26",
+      "file_length": "4096",
+      "file_date": "Apr 20 2017 23:42:59.0000000000 +00:00",
+      "file_name": "/bootflash/.rollback_timer"
+    },
+    "27": {
+      "file_index": "27",
+      "file_length": "30",
+      "file_date": "May 02 2018 07:35:49.0000000000 +00:00",
+      "file_name": "/bootflash/throughput_monitor_params"
+    },
+    "28": {
+      "file_index": "28",
+      "file_length": "457445528",
+      "file_date": "May 01 2018 04:25:39.0000000000 +00:00",
+      "file_name": "/bootflash/isr4300-universalk9.16.03.03.SPA.bin"
+    }
+  },
+  "bytes_available": "13298044928",
+  "bytes_used": "946311168"
```

Comment:
same with other comments. if always number, we use integer for numbers.

Reply:
change numbers to integer and removed duplicate index